src/CosmosDB/CosmosDB.Test/ScenarioTests/RestoreTests.cs

```diff
@@ -134,5 +134,12 @@ public void TestProvisionCosmosDBAccountBackupPolicyWithContinuous7DaysCmdLets()
         {
             TestRunner.RunTestScript("Test-ProvisionCosmosDBAccountBackupPolicyWithContinuous7DaysCmdLets");
         }
+
+        [Fact]
+        [Trait(Category.AcceptanceType, Category.CheckIn)]
+        public void TestCrossRegionRestoreAccountCmdlets()
+        {
+            TestRunner.RunTestScript("Test-CrossRegionRestoreAccountCmdlets");
```

Comment:
please add one more test for single region cross region restore

Reply:
added and got a clean run